src/target/riscv/riscv-013.c

```diff
@@ -1615,11 +1615,7 @@ static int examine(struct target *target)
 		LOG_ERROR("Debugger is not authenticated to target Debug Module. "
 				"(dmstatus=0x%x). Use `riscv authdata_read` and "
 				"`riscv authdata_write` commands to authenticate.", dmstatus);
-		/* If we return ERROR_FAIL here, then in a multicore setup the next
-		 * core won't be examined, which means we won't set up the
-		 * authentication commands for them, which means the config script
-		 * needs to be a lot more complex. */
-		return ERROR_OK;
+		return ERROR_FAIL;
```

Comment:
Did this change come from the RISC-V community?  Just wondering if the issue mentioned in the comment that used to be there (saying that returning ERROR_FAIL could be problematic for multi-core, and ERROR_OK was being returned for that reason) is no longer an issue.